packages/env/README.md

```diff
@@ -201,6 +201,14 @@ ID      user_login      display_name    user_email      user_registered roles
 ✔ Ran `wp user list` in 'cli'. (in 2s 374ms)
 ```
 
+### `docker logs -f [container] >/dev/null` 
+
+```sh
+docker logs -f <container> >/dev/null 
```

Comment:
Could we clarify that `container` here would be the ID of docker container, which is visible with `docker ps -a`?

We could merge this in the meantime to hold us over until we add proper log support 👍

Reply:
Have updated to make it clearer that it is the container id that is needed.